include/pisa/scorer/bm25.hpp

```diff
@@ -9,24 +9,25 @@ namespace pisa {
 
 template <typename Wand>
 struct bm25: public index_scorer<Wand> {
-    static constexpr float b = 0.4;
-    static constexpr float k1 = 0.9;
-
     using index_scorer<Wand>::index_scorer;
 
-    static float doc_term_weight(uint64_t freq, float norm_len)
+    bm25(const Wand& wdata, const float b, const float k1)
```

Comment:
I was wondering, maybe we can at least put a link reference to some description of BM25 for explanation of `b` and `k1`. I imagine for someone just coming to see the code, it's pretty confusing.